external-builds/pytorch/repo_management.py

```diff
@@ -192,7 +192,33 @@ def repo_hashtag_to_patches_dir_name(version_ref: str) -> str:
     return version_ref
 
 
-def do_checkout(args: argparse.Namespace):
+def do_hipify(args: argparse.Namespace):
+    repo_dir: Path = args.repo
+    print(f"Hipifying {repo_dir}")
+    build_amd_path = repo_dir / "tools" / "amd_build" / "build_amd.py"
+    if os.path.exists(build_amd_path):
+        exec([sys.executable, build_amd_path], cwd=repo_dir)
```

Comment:
edit: oh, this was moved from preexisting code. Fine to ignore, or accept the cleanup

Any particular reason to use `os.path.exists` here over pathlib? https://docs.python.org/3/library/pathlib.html#corresponding-tools

```suggestion
    if build_amd_path.exists():
        exec([sys.executable, build_amd_path], cwd=repo_dir)
```